aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseAPI/Models/DateRange.cs

```diff
@@ -1,36 +1,35 @@
 namespace BindTryParseAPI.Models
 {
     // <snippet>
-    public class DateRange
+    public class DateRange : IParsable<DateRange>
     {
-        public DateOnly? From { get; }
-        public DateOnly? To { get; }
+        public DateOnly? From { get; set; }
+        public DateOnly? To { get; set; }
```

Comment:
```suggestion
        public DateOnly? To { get; init; }
```

Or keep it readonly by making it private?

Reply:
Maybe keep the ctor and not change the properties is another option.